### ISSUE TITLE
Compiler Error Message: CS0234: The type or namespace name 'Ajax' does not exist in the namespace 'System.Web.Mvc' (are you missing an assembly reference?)

### ISSUE THREAD

**Issue by papyr** [TypecastException/AspNetIdentity2GroupPermissions#4]
Looks like its a little old, 

Server Error in '/' Application.

Compilation Error

Description: An error occurred during the compilation of a resource required to service this request. Please review the following specific error details and modify your source code appropriately. 

Compiler Error Message: CS0234: The type or namespace name 'Ajax' does not exist in the namespace 'System.Web.Mvc' (are you missing an assembly reference?)

Source Error:

Line 22:     using System.Web.WebPages;
Line 23:     using System.Web.Mvc;
Line 24:     using System.Web.Mvc.Ajax;
Line 25:     using System.Web.Mvc.Html;
Line 26:     using System.Web.Optimization;

**Comment by winecraft** [comment]
I have the same error in my web config I removed the namespaces, it compiled, then broke when I did a nuget update

**Comment by xivSolutions** [comment]
Yeah, this was put together in the early days of Identity 2.0. I will update as soon as I can - I'm moving cross-country for the next few days though. If one of you fine gents wants to update and shoot me a PR, I'm happy to merge it in. Otherwise, it may be a few days at least...

Thanks for taking the time to comment here though!
